lib/tasks_private/instance_types.rake

```diff
@@ -5,9 +5,6 @@ task :instance_types => [:environment] do
   # This task creates a instance_types.rb file in the current directory based upon the current
   # instance_types.rb file and data from
   #  curl https://raw.githubusercontent.com/powdahound/ec2instances.info/master/www/instances.json > lib/tasks_private/instance_types_data/instances.json
-  #  https://aws.amazon.com/ec2/instance-types/#instance-type-matrix > lib/tasks_private/instance_types_data/instance_types.csv
```

Comment:
I see that this is now included in ec2instances.info, which is great

Reply:
Yes, they just merged my https://github.com/powdahound/ec2instances.info/pull/270 hours ago 😉